hugo_site/content/copenhagen.md

```diff
@@ -36,17 +36,17 @@ To know which line you need to jump in, you can use either [Google Maps](https:/
 
 As you might know, Denmark's official currency is the Danish crown (Danske Kroner, a.k.a `DKK`). Some shops might accept Euro in cash as payment, but this is not a rule. 
 
-One nice convenience is the fact that most establishments in Denmark (hotels, bars, restaurants, shops) will accept payment with Credit Card (VISA and MasterCard). As a matter of fact, shops are allowed not to accept cash should they judge convinient, however in very touristic place this is not a common practice. So should you want practicity, just credit cards instead of cash.
+One nice convenience is the fact that most establishments in Denmark (hotels, bars, restaurants, shops) will accept payment with Credit cards (VISA and MasterCard). As a matter of fact, shops are allowed not to accept cash should they judge convinient, however in most touristic places this is not a common practice. So should you want practicity, just use your credit cards instead of cash. This way you will avoid having to carry cash around and exchange it all the time.
```

Comment:
I have to say that this is a bit biased :) I know that a lot of people like credit cards, but I prefer cash for most things. The law still states that you need an actual exception to reject cash.

```suggestion
Most establishments in Denmark (hotels, bars, restaurants, shops) accept payment with credit cards (VISA and MasterCard). There are even places that reject cash (however such practice isn't allowed in established places). Many people will find it more practical to use credit cards instead of cash, but the Danish Kroner still retain their charm and usefulness.
```

Reply:
I had read [somewhere](https://www.fm.dk/temaer/vaekst-2015/delvis-afskaffelse-af-kontantreglen) that the government was working to remove the obligation to pay with cash in some establishments. Since this is only a forslag, I don't know if this actually came into pratice. 

I'll update that part.